README.md

```diff
@@ -1,49 +1,27 @@
-# Shopinvader getting
+# Shopinvader getting started
 
+Get a local demo of Shopinvader in a minimum of steps.
 
-## With docker-compose
+## Requirements
 
-This cmd will run locomotive + odoo in background
+docker-compose https://docs.docker.com/compose/install/
+git (optional, you can manually download https://raw.githubusercontent.com/akretion/shopinvader-getting-started/master/docker-compose.yml)
```

Comment:
s/optional, you can manually download/optionally, you can instead manually download/